src/components/grid-aware/ProgramBlock/ProgramBlock.module.css

```diff
@@ -0,0 +1,139 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: var(--bleed-grid-template-columns);
+  margin: auto;
+  margin-bottom: 100px;
+  max-width: var(--bleed-max-width);
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.bleedWrapper > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+    margin-bottom: 80px;
+  }
+}
+
+.bleedMainContent {
+  grid-column: 2;
+}
+
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+}
+
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: flex;
+    flex-direction: column-reverse;
+  }
+}
+
+
+/* Main grid areas */
+
+.gridAreaText {
+  grid-column: 6/ span 6;
+  grid-row: 1;
+}
+
+@media (--tablet-and-down) {
+  .gridAreaText {
+    padding-top: 16px;
+  }
+}
+
+.gridAreaImage {
+  grid-column: 2 / span 4;
+  grid-row: 1;
+}
+
+/* Left grid area */
+
+.title {
+  font: var(--font-body-large);
+  margin-bottom: 5px;
+  margin-top: 0;
+}
+
+@media (--tablet-and-down) {
+  .title {
+    margin-top: 0;
+  }
+}
+
+.subtitle {
+  font: var(--font-subtitle);
+}
+
+.description {
+  font: var(--font-body-medium);
+  margin-top: 20px;
+}
+
+.description :first-child {
+  margin: 0;
+}
+
+.description > :not(:first-child) {
+  margin-top: 20px;
+}
+
+.description ul {
+  margin-left: 1.5em;
+}
+
+.description ul > li {
+  list-style-position: outside;
+  list-style-type: disc;
+}
+
+.ctaButtonRow {
+  display: flex;
+  flex-flow: row wrap;
+  margin-top: 30px;
+}
+
+.ctaButtonRowItem:not(:last-child) {
+  margin-right: 30px;
+}
+
+@media (--tablet-and-down) {
+  .ctaButtonRow {
+    flex-flow: column;
+  }
+
+  .ctaButtonRowItem:not(:last-child) {
+    margin-bottom: 20px;
+    margin-right: 0;
+  }
+}
+
+
+/* Right grid area*/
+
+.image {
+  height: 250px;
+  width: 100%;
```

Comment:
These two properties by themselves are going to stretch and distort the images. You'll want to add [`object-fit: cover`](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit) so that the images are cropped to fit instead of stretched.